.env.example

```
@@ -1,3 +1,4 @@
 POSTGRES_PASSWORD=password
 DATABASE_URL=postgres://postgres:$POSTGRES_PASSWORD@localhost:5432
 GOVUK_NOTIFY_API_KEY=
+SIGN_IN_METHOD=
```

As it isn't a secret, I think you can go ahead and say
```suggestion
SIGN_IN_METHOD=persona
```

Just reduces the number of steps for getting your local dev environment up and running

---

resolved in [9631b56](https://github.com/DFE-Digital/itt-mentor-services/pull/9/commits/9631b56b044063ffd9f0187cc6c3645fe1114bc0)